src/error.rs

```diff
@@ -508,6 +531,7 @@ pub struct WriteError {
 #[non_exhaustive]
 pub struct BulkWriteError {
     /// Index into the list of operations that this error corresponds to.
+    #[serde(default = "default_index")]
```

Comment:
this is required since one of the tests omits this field, filed DRIVERS-1637 for updating it.

Reply:
`Default` is already implemented for `usize` to be zero, so I think this can just be `#[serde(default)]` (without needing a custom function).

Reply:
Oh nice, I didn't know that, fixed.